Bottom: 9a87e2bb1e5d76a515e2eaf6536c47201c6328df
Top:    1cb9eeabb985a41a33674e4e194f6163065ce525
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-03 23:51:29 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
